PurchasesCoreSwift/Networking/ETagManager.swift

```diff
@@ -0,0 +1,184 @@
+//
+// Created by César de la Vega on 4/16/21.
+//
+
+import Foundation
+
+internal let ETAG_HEADER_NAME: String = "X-RevenueCat-ETag"
+
+@objc(RCETagManager) public class ETagManager: NSObject {
+    private let queue = DispatchQueue(label: "ETagManager")
+
+    private var userDefaults: UserDefaults
+
+    @objc public override init() {
+        // TODO: do they persist after reinstalling
+        let bundleID: String = Bundle.main.bundleIdentifier ?? ""
```

Comment:
I ended up doing this @aboedo but I am not sure it's correct, and it wasn't very clear in the docs either.